server/mediapublic/mediapublic/__init__.py

```diff
@@ -19,48 +19,57 @@ def main(global_config, **settings):
 
     # INDEX
     #config.add_route('index', '/')
-    
+
     # ABOUT
     #config.add_route('about', '/')
-    
+
     # USERS
     config.add_route('users', '/users')
     config.add_route('user_by_id', '/users/:{id}')
-    
+
     # RECORDING CATEGORIES
     config.add_route('user_types', '/user_types')
     config.add_route('user_type_by_id', '/user_types/:{id}')
-    
+
     # RECORDING CATEGORIES
     config.add_route('recording_categories', '/recording_categories')
     config.add_route('recording_category_by_id', '/recording_categories/:{id}')
-    
+
     # ORGANIZTIONS
     config.add_route('organizations', '/organizations')
     config.add_route('organization_by_id', '/organizations/:{id}')
     config.add_route('organizations_comments', 'organizations/:{oid}/comments')
     config.add_route('organizations_comment_by_id', 'organizations/:{oid}/comments/:{id}')
-    
+
     # PEOPLE
     config.add_route('people', '/people')
     config.add_route('organization_people', '/organizations/:{oid}/people')
     config.add_route('organization_people_by_id', 'organizations/:{oid}/people/:{id}')
     config.add_route('organization_people_comments', 'organizations/:{oid}/people/:{pid}/comments')
     config.add_route('organization_people_comment_by_id', 'organizations/:{oid}/people/:{pid}/comments/:{id}')
-    
+
     # RECORDINGS
     config.add_route('recordings', '/recordings')
     config.add_route('organization_recordings', '/organizations/:{oid}/recordings')
     config.add_route('organization_recordings_by_id', 'organizations/:{oid}/recordings/:{id}')
     config.add_route('organization_recordings_comments', 'organizations/:{oid}/recordings/:{rid}/comments')
     config.add_route('organization_recordings_comment_by_id', 'organizations/:{oid}/recordings/:{rid}/comments/:{id}')
-    
+
+    # PLAYLISTS
+    config.add_route('playlists', '/playlists')
+    config.add_route('user_playlists', '/users/:{uid}/playlists')
+    config.add_route('new_user_playlist', '/users/:{uid}/playlists/new')
```

Comment:
These should have the same naming convention as the other end-points.  So this one would be `user_playlists_new`.  Same thing for likes 62 - 65.